src/platform/Kconfig

```diff
@@ -385,6 +385,13 @@ config CAVS_USE_LPRO_IN_WAITI
 	  After waiti exit clock source will be restored.
 	  Choose n if unclear.
 
+config KCPS_DYNAMIC_CLOCK_CONTROL
```

Comment:
should it depend on IPC4? Because, e.g., it seems to assume that a PAUSE always precedes a STOP, which isn't the case with IPC3?

Reply:
Yes, it should